### PR TITLE
Closes #60 Allow Teacher to Unpair Students

### LINKED ISSUE #60: Ability to unpair students chat

### PR DESCRIPTION
**Pull request by zachlankton**
Closes #60 Allow Teacher to Unpair Students
Related Backend PR https://github.com/mssiegel/frempco-server/pull/12

This PR allows the teacher to unpair students from their dashboard using an unpair button as shown in the image below.

unpairStudents

### CODE REVIEW

File: src/components/pages/TeachersPage/index.tsx [hunk 1]
```diff
@@ -43,6 +43,14 @@ export default function TeachersPage({ classroomName }: ClassroomProps) {
     // },
   ]);
 
+  const unPair = (chatId, student1, student2) => {
+    const unpairConfirmed = confirm(
```
Comment (mssiegel) [inline src/components/pages/TeachersPage/index.tsx:47]: Great usage of `window.confirm()`!! Engineering time is everything and using a prebuilt `confirm` function saves us a lot of time!

File: src/components/pages/TeachersPage/index.tsx [hunk 2]
```diff
@@ -141,6 +157,7 @@ export default function TeachersPage({ classroomName }: ClassroomProps) {
             studentChats={studentChats}
             setDisplayedChat={setDisplayedChat}
             displayedChat={displayedChat}
+            unPair={unPair}
```
Comment (mssiegel) [inline src/components/pages/TeachersPage/index.tsx:160]: Throughput this page, unpair was unpair.  So P wasn't capitalized. Let's keep it that way for consistency.
```suggestion
            unpair={unpair}
```

File: src/components/pages/TeachersPage/PairedStudentsList.tsx [hunk 3]
```diff
@@ -53,12 +54,23 @@ export default function PairedStudentsList({
...
-                      Display chat
-                    </Button>
+                    <Box sx={{ paddingBottom: 1 }}>
+                      <Button
+                        size='small'
+                        onClick={() => setDisplayedChat(chatId)}
+                      >
+                        Display chat
+                      </Button>
+
```
Comment (mssiegel) [inline src/components/pages/TeachersPage/PairedStudentsList.tsx:64]: The buttons are very close together. Can we add some margin between them?
image

**Review by mssiegel — APPROVED** [review]
Fantastic PR! I left a few minor comments to be addressed.  Once you do so, feel free to go ahead and merge without another need for approval.
The backend PR was already approved.

File: src/components/pages/TeachersPage/PairedStudentsList.tsx (hunk 3 above)
Comment (zachlankton) [reply src/components/pages/TeachersPage/PairedStudentsList.tsx:64]: I updated this to look like so:

betterspacing

**Pull request merged**